compiler/optimizations/gpuTransforms.cpp

```diff
@@ -538,7 +545,10 @@ void GpuKernel::populateBody(CForLoop *loop, FnSymbol *outlinedFunction) {
                 INT_FATAL("Malformed PRIM_GET_MEMBER_*");
               }
             }
-            else if (parent->isPrimitive()) {
+           else if(parent->isPrimitive(PRIM_GPU_SET_BLOCKSIZE)) {
+              // This is handled at kernel launch time, don't pass into the kernel
+           }
```

Comment:
I would like to believe that if we do one of the things I suggested above, we won't need this branch. If we still need this branch, *maybe* we should add `copyNode=false` here as well, just to make the code more self-documenting?

Reply:
I removed it (Fixed in 9d559a8ac3b726ecaf63d09af50ae5ddac2eeafc)